From 0074f59b51a2882ce3b8e3922c52f4c547bd56b7 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 10 Mar 2011 16:48:36 -0800
Subject: usb-storage: ene_ub6250 depends on USB_STORAGE
Patch-mainline: v2.6.39
Git-commit: 0074f59b51a2882ce3b8e3922c52f4c547bd56b7

Fix ene_ub6250 build: it uses usb_storage driver interfaces, so it
should depend on USB_STORAGE.

ene_ub6250.c:(.text+0x14ff19): undefined reference to `usb_stor_reset_resume'
ene_ub6250.c:(.text+0x14ffb1): undefined reference to `usb_stor_bulk_transfer_buf'
ene_ub6250.c:(.text+0x14ffdd): undefined reference to `usb_stor_bulk_srb'
ene_ub6250.c:(.text+0x14fff1): undefined reference to `usb_stor_bulk_transfer_sg'
ene_ub6250.c:(.text+0x1503dd): undefined reference to `usb_stor_set_xfer_buf'
ene_ub6250.c:(.text+0x15048e): undefined reference to `usb_stor_access_xfer_buf'
ene_ub6250.c:(.text+0x150723): undefined reference to `usb_stor_probe1'
ene_ub6250.c:(.text+0x150795): undefined reference to `usb_stor_probe2'
ene_ub6250.c:(.text+0x1507af): undefined reference to `usb_stor_disconnect'
drivers/built-in.o:(.data+0x10224): undefined reference to `usb_stor_suspend'
drivers/built-in.o:(.data+0x10230): undefined reference to `usb_stor_pre_reset'
drivers/built-in.o:(.data+0x10234): undefined reference to `usb_stor_post_reset'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/storage/Kconfig
+++ b/drivers/usb/storage/Kconfig
@@ -212,6 +212,7 @@ config USB_LIBUSUAL
 config USB_STORAGE_ENE_UB6250
 	tristate "USB ENE card reader support"
 	depends on USB && SCSI
+	depends on USB_STORAGE
 	---help---
 	  Say Y here if you wish to control a ENE SD Card reader.
 	  To use SM/MS card, please build driver/staging/keucr/keucr.ko
